python_bitvavo_api/bitvavo.py

```diff
@@ -96,6 +96,8 @@ def __init__(self, reset, bitvavo):
     threading.Thread.__init__(self)
 
   def waitForReset(self, waitTime):
+    if waitTime < 0.0:
+      waitTime = 0.001  # 1ms
     time.sleep(waitTime)
```

Comment:
I think we should skip sleeping altogether when the wait time is zero or negative:
```suggestion
    if waitTime > 0.0:
      time.sleep(waitTime)
```